Update documentation to specify how long it takes for exceptions to reach the endpoint (#2959)
elastic/security-docs: docs/detections/add-exceptions.asciidoc

* *Close all alerts that match this exception and were generated by this rule*:
Closes all alerts that match the exception's conditions.

. Click *Add Endpoint Exception*. An exception is created for both the detection rule and the {elastic-endpoint}. 
+ 

> NOTE: The speed at which the exception is applied to {elastic-endpoint} depends on the size of your deployment. It may take longer for exceptions to be applied to hosts within larger deployments.

Author: @kevinlog this note describes some of what we were discussing and warns users that exceptions can take some time to apply to hosts in larger deployments. If this is not the kind of messaging we want to convey, let me know and I can remove the note.

Reviewer: @nastasha-solomon this looks good to me.  I think it's OK to give users a heads up that something may take longer, but we should avoid a timeframe

Reviewer 2: Both sentences kind of say the same thing, so maybe we just need one of them:
```suggestion
NOTE: It might take longer for exceptions to be applied to hosts within larger deployments.
```

Author: ^ This is the way. I'll merge your edits - thanks, @joepeeples !
